### PR TITLE
Upgrade to 1.16.0

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix linter warnings
- Upgrade to 1.16.0

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -18,9 +18,8 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
-lang=$(ynh_app_setting_get --app=$app --key=lang)
+language=$(ynh_app_setting_get --app=$app --key=language)
```
Comment (yalh76) [inline scripts/upgrade:22]: You need to manage the var name change...
If not upgrade from previous version will fail

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: scripts/upgrade [hunk 2]
```diff
@@ -45,13 +45,19 @@ if [ -z "$final_path" ]; then
 	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 fi
 
+# If language doesn't exist, create it
+if [ -z "$language" ]; then
+	language="en"
+	ynh_app_setting_set --app=$app --key=language --value=$language
+fi
```
Comment (yalh76) [inline scripts/upgrade]: perfect, now would be a good idea to remove old `lang` var

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: scripts/upgrade (hunk 2 above)
Comment (ericgaspar) [reply scripts/upgrade]: mmh ... I don't see where I left `lang`var behind ... What exactly should I remove?

Comment (yalh76) [reply scripts/upgrade]: the `lang` var is in the settings of previous installed rainloop, when they will be updated, as you change the var name, you should remove it

Comment (ericgaspar) [reply scripts/upgrade]: how do I do that?

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade (hunk 2 above)
Comment (yalh76) [reply scripts/upgrade]: You can use: `ynh_app_setting_delete`

Comment (ericgaspar) [reply scripts/upgrade]: Thanks

**Pull request merged**